modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -1252,6 +1252,16 @@ private void _assertPagination(
 		JSONArray itemsJSONArray = jsonObject.getJSONArray("items");
 
 		Assert.assertEquals(2, itemsJSONArray.length());
+
+		Set<Long> ids = new HashSet<>();
+
+		for (Object item : itemsJSONArray) {
+			JSONObject itemJSONObject = (JSONObject)item;
+
+			ids.add(itemJSONObject.getLong("id"));
+		}
+
+		Assert.assertTrue(ids.contains(baseModel.getPrimaryKeyObj()));
```

Comment:
Just a resume about what we discussed:
- Pass the list of baseModel (it is always 2 items);
- Add test for Pagination 2;
- Add `Assert.assertEquals(1, jsonArray.length());` for the pagination 1 and 2;
- Compare the item of each pagination with the baseModel list;
- Option1: You can compare and remove the item from the List and Assert that the list is empty.
- Option2: After both pagination tests (pagination 1 and 2), you can compare the JSONArray using `JSONAssert.assertEquals()`.

Reply:
Done! Please let me know if I've missed anything